lib/fsext/changepathfs.go

```diff
@@ -52,7 +52,7 @@ func (f *ChangePathFile) Name() string {
 }
 
 // Chown changes the uid and gid of the named file.
-func (b *ChangePathFs) Chown(name string, uid, gid int) error {
+func (b *ChangePathFs) Chown(_ string, _, _ int) error {
 	return errors.New("unimplemented Chown")
```

Comment:
I would like to have a panic here

Reply:
[327040d](https://github.com/grafana/k6/pull/3482/commits/327040d3561e10a695247afd4aa57ca5567a9075)